docs/experiments/overview.md

```diff
@@ -117,3 +97,33 @@ For example:
 ## Generic pattern matching
 
 See [generic pattern matching](../generic-pattern-matching/).
+
+## Deprecated experiments
+
+### Equivalences
+
+:::note
+This feature was deprecated in Semgrep v0.61.0.
```

Comment:
Is there a reason we can give for this deprecation?

Reply:
I’m not aware. @brendongo any ideas?

Reply:
https://r2c-community.slack.com/archives/CK86BJ5DW/p1628577588061300

I believe there is an issue or something somewhere in Semgrep's GitHub project. But to my knowledge the feature was not being used that much, and the really important equivalences were being made built-in, so it was proposed to deprecate it in order to focus on other things. Besides, now with the `options:` key is much easier to add equivalences because the users have the power to enable/disable them if needed, previously you could not so we had to be more careful with what equivalences we added.